resources/shared_translations.txt

```diff
@@ -4191,6 +4191,10 @@
 "operations.default-description" "en" "<p>The procedure you have described usually requires a permit or a notification.</p><p>The requirement for a permit is disclosed in the municipality's own construction ordinance.</p><p>If you cannot find the information you require, you may verify the instructions for transactions by making an advice request.</p><p>Your question will be forwarded to a municipal authority responsible for permit matters.</p><p>You will be notified by e-mail, once a response has been received.</p>"
 "operations.default-description" "fi" "<p>Kuvaamasi toimenpide vaatii yleensä luvan tai ilmoituksen.</p><p>Luvanvaraisuudesta kerrotaan muun muassa kunnan omassa rakennusjärjestyksessä.</p><p>Mikäli et löydä tarvitsemaasi tietoa, voit varmistaa asiointiin liittyvät ohjeet tekemällä neuvontapyynnön.</p><p>Kysymyksesi välitetään kunnan lupa-asioista vastaavalle viranomaiselle.</p><p>Lähetämme sinulle tiedon saapuneesta vastauksesta sähköpostiviestin välityksellä.</p>"
 "operations.default-description" "sv" "<p>Åtgärden du beskriver kräver i allmänhet tillstånd eller anmälan. Tillståndsplikten behandlas bland annat i kommunens egen byggnadsordning.</p><p>Om du inte hittar information som du behöver, kan du kontrollera anvisningar som gäller ärendehantering  genom att göra begäran om rådgivning.</p><p>Din fråga förmedlas till myndigheten som är ansvarig om tillståndsärenden i din kommun. Vi informerar dig om svaret per e-post.</p>"
+"operations.erityista-toimintaa-alue" "fi" "Erityistä toimintaa varten rakennettava alue"
+"operations.erityista-toimintaa-alue.desc" "en" "<p>The procedure you have described usually requires a permit or notification.</p><p>The requirement for a permit is disclosed in the municipality's own construction ordinance.</p><p>If you cannot find the information you require, you may verify the instructions for transactions by making an advice request.</p><p>Your question will be forwarded to a municipal authority responsible for permit matters.</p><p>You will be notified by e-mail, once a response has been received.</p><p>"
```

Comment:
Englannilla ei niin väliä, mutta ehkä mieluummin:
information you ~require~ need

Reply:
Mä en uskalla korjata kun nää on kopipastea muista vastaavista. Plus Jessen kanssa oli puhetta, että nää kaikki menee uusiks enivei kun tulee se uusi asiointipolku 🧙